.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,78 @@
+---
+name: "Bug Report"
+description: "Report a bug to help us improve."
+title: "[Bug]: <title>"
+type: "Bug"
+body:
+  - type: markdown
+    attributes:
+      value: "Please fill out the following details to help us address the issue."
+  - type: input
+    id: title
+    attributes:
+      label: "Bug Title"
+      description: "Provide a short and descriptive title for the bug."
+      placeholder: "e.g., App crashes on startup"
```

App crashing is unlikely to be something that happens, assuming that "app" here is likely to be interpreted as the IDE/agent people are using.

```suggestion
      placeholder: "e.g., can't run an aggregation"
```